pkg/operator/starter.go

```diff
@@ -654,3 +615,38 @@ func singleNameListOptions(name string) func(opts *metav1.ListOptions) {
 		opts.FieldSelector = fields.OneTermEqualSelector("metadata.name", name).String()
 	}
 }
+
+func apiServices() []*apiregistrationv1.APIService {
+	var apiServiceGroupVersions = []schema.GroupVersion{
+		// these are all the apigroups we manage
+		{Group: "oauth.openshift.io", Version: "v1"},
+		{Group: "user.openshift.io", Version: "v1"},
+	}
+
+	ret := []*apiregistrationv1.APIService{}
+	for _, apiServiceGroupVersion := range apiServiceGroupVersions {
+		obj := &apiregistrationv1.APIService{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: apiServiceGroupVersion.Version + "." + apiServiceGroupVersion.Group,
+				Annotations: map[string]string{
+					"service.alpha.openshift.io/inject-cabundle":   "true",
+					"authentication.operator.openshift.io/managed": "true",
```

Comment:
I think we also don't need this annotation. It was used in OAS-O to check if it is safe to stop managing an API group. 
I think that we have at least two options:

`Option 1`: we remove it in 4.7 then:

In 4.6 the fight is over and CAO is responsible for these groups.
In 4.7 nothing changes.

An Upgrade from 4.6 to 4.7 will work
A downgrade from 4.7 to 4.6 might be a bit disruptive since CAO and OSA-O needs to decide which one is managing. 

`Option 2`: we remove it in 4.8 then a downgrade from 4.7 to 4.6 is smoother, no disruption.